packages/react-dom/src/client/ReactDOMFiberInput.js

```diff
@@ -244,6 +244,10 @@ export function postMountWrapper(
   }
   node.defaultChecked = !node.defaultChecked;
   node.defaultChecked = !node.defaultChecked;
+  // Set the "checked" attribute initially.
+  if (props.hasOwnProperty('defaultChecked') && !!props.defaultChecked) {
+    node.setAttribute('checked', 'checked');
+  }
```

Comment:
I wonder if lines 245-250 could be replaced with:

```javascript
node.defaultChecked = node._wrapperState.initialChecked
```

Reply:
You are right. The ```node._wrapperState.initialChecked``` will simply the DOM operations.